### ISSUE TITLE
Make contact mandatory field when creating organisations

### ISSUE THREAD

**Issue by ManonGros** [gbif/registry#139]
Right now, registry editors and admins can create new organisations without contacts. However, this is a mandatory field in the https://www.gbif.org/become-a-publisher form.
Could we make it a mandatory field in the registry as well?
Thanks!

**Comment by MortenHofft** [comment]
The current UI is made to reflect the APIs as closely as possible. There is no restrictions on the number of contacts in the backend. Since we have users that create using the APIs, we should probably change it there if we believe it is essential.

It isn't entirely clear to me how to add a restriction in the frontend (only or as well) in a good way.

Perhaps simply highlighting issues such as missing contacts, invalid coordinates etc. in search results and on the detail pages could do? At least it would make it clear to editors that there is a potential issue with the content. 

Or alternatively we could make it mandatory to fill the organisation email (not contacts)

**Comment by MortenHofft** [comment]
I suddenly remember something.

Making it mandatory of course will only work if all existing data has the data already.

To begin with we had more mandatory fields, but then you @ManonGros noticed that it was frustrating that you could not edit an entity without also fixing issues such as missing emails. So we removed it again. We are repeating ourselves 😄

**Comment by ManonGros** [comment]
True! And I agree it is frustrating, but if it is mandatory in the registration form, shouldn't it be also mandatory in the registry?
This is a screenshot of the registration form, any idea where this gets stored?
![Screenshot 2019-08-13 at 13 50 15](https://user-images.githubusercontent.com/7677271/62939405-7decdb80-bdd1-11e9-8112-71f85dbd225d.png)

Alternatively, I think that having a warning would also work.

**Comment by MortenHofft** [comment]
I do not like it as a fix, but I guess what you would like is:

if we could have it as a requirement when you create an organisation and when you edit organisations that already the information. But not have it as a requirement if it is an existing organisation without it.

I'm not keen on it. I think it is ugly. And more importantly it is cumbersome to change with the current implementation 😄. But that of course should not be the argument. But it isn't a quick fix I suspect.

**Comment by ManonGros** [comment]
In that case, how about just a warning if the email for the organization is missing?

I don't want to make it more cumbersome than it has to be. This idea is to at least remind users to put a contact somewhere.
